src/main/webapp/WEB-INF/jsp/feedback_form.jsp

```diff
@@ -51,11 +52,23 @@
             <p>
             <input aria-label="send button" type="submit" value="Send" class='btn btn-primary btn-share-ideas' />
             </p>
-        </form>
+        
       </div>
       <div class="col-xs-12 col-sm-6 general-help">
-        <jsp:directive.include file="/WEB-INF/jsp/include-righthand-links.jsp"/>
```

Comment:
It's still used on the success page. I'm in the process of converting those links to read the portlet preferences.